src/components/ChangePassword/index.js

```diff
@@ -0,0 +1,186 @@
+`;
+
+export default function ChangePassword() {
+  Component.call(this, { html, events });
+  const $changePasswordForm = this.selected.get('change-password-form');
+  const $passwordInputContainer = this.selected.get('password-input-container');
+  const $passwordErrorMessage = this.selected.get('password-error-message');
+  const $newPasswordInputContainer = this.selected.get(
+    'new-password-input-container',
+  );
+  const $newPasswordErrorMessage = this.selected.get(
+    'new-password-error-message',
+  );
+  const $confirmPasswordInputContainer = this.selected.get(
+    'confirm-password-input-container',
+  );
+  const $confirmPasswordErrorMessage = this.selected.get(
+    'confirm-password-error-message',
+  );
+
+  const passwordInput = new TextInput({
+    placeholder: 'Senha',
+    assetUrl: eyeSlashIcon,
+    assetPosition: 'suffix',
+  });
+  const newPasswordInput = new TextInput({
+    placeholder: 'Nova senha',
+    assetUrl: eyeSlashIcon,
+    assetPosition: 'suffix',
+  });
+  const confirmPasswordInput = new TextInput({
+    placeholder: ' Confirmar senha',
+    assetUrl: eyeSlashIcon,
+    assetPosition: 'suffix',
+  });
+  const submitButton = new Button({
+    text: 'Salvar',
+    isFullWidth: true,
+    isDisabled: true,
+  });
+
+  passwordInput.mount($passwordInputContainer);
+  newPasswordInput.mount($newPasswordInputContainer);
+  confirmPasswordInput.mount($confirmPasswordInputContainer);
+  submitButton.mount($changePasswordForm);
+
+  passwordInput.selected.get('input-text').type = 'password';
+  newPasswordInput.selected.get('input-text').type = 'password';
+  confirmPasswordInput.selected.get('input-text').type = 'password';
+
+  const validateFields = () => {
+    const password = passwordInput.selected.get('input-text').value;
+    const newPassword = newPasswordInput.selected.get('input-text').value;
+    const confirmPassword =
+      confirmPasswordInput.selected.get('input-text').value;
+
+    if (password && newPassword && confirmPassword) {
+      submitButton.enable();
+    } else {
+      submitButton.disable();
+    }
+  };
+
+  passwordInput.selected
+    .get('input-text')
+    .addEventListener('input', validateFields);
+  newPasswordInput.selected
+    .get('input-text')
+    .addEventListener('input', validateFields);
+  confirmPasswordInput.selected
+    .get('input-text')
+    .addEventListener('input', validateFields);
+
+  submitButton.listen('click', () => {
+    const password = passwordInput.selected.get('input-text').value;
+    const newPassword = newPasswordInput.selected.get('input-text').value;
+    const confirmPassword =
+      confirmPasswordInput.selected.get('input-text').value;
+    let validPassword = true;
+    let validNewPassword = true;
+    let validConfirmPassword = true;
+
+    if (!this.validatePassword(password)) {
+      validPassword = false;
+      $passwordErrorMessage.classList.add('show-error');
+      $passwordErrorMessage.innerText = 'Senha inválida';
```

Comment:
O texto `Senha inválida` pode ser colocado direto no `span` no html, não precisa colocar o conteúdo apenas quando ele for aparecer! Ai basta esconder com CSS e exibir quando aplicar a classe show-error, ela teria a unica responsabilidade de dar um `display: block`

Isso vale pra todos os erros!